### ISSUE TITLE
Notes added to default notes ref are picked up by git-ratchet and cause failures

### ISSUE THREAD

**Issue by marcins** [iangrunert/git-ratchet#5]
Steps to reproduce:

```
git init .
echo "Hello">README.txt
git add README.txt && git commit -m"Test"
echo "test,1"|git ratchet check -vw
git-ratchet dump # works fine
git notes add -m"Hello"
git-ratchet dump # fails
FATAL: 2015/06/30 Badly formatted measures
```

It looks like the command `git --no-pager log --show-notes=git-ratchet-1-master --pretty=format:'%H,%an <%ae>,%at,"%N",' HEAD` picks up notes from the `git-ratchet-1-master` ref AND the default ref.

According to http://git-scm.com/docs/git-show `--show-notes` is deprecated and `--notes` should be used - doing `git --no-pager log --notes=git-ratchet-1-master --pretty=format:'%H,%an <%ae>,%at,"%N",' HEAD` shows the correct notes.

**Comment by iangrunert** [comment]
Thanks for the bug report!

I think I've fixed this on master with 4096e587f86abb4532750883d9f0a69bb9392685 . I'll try and cut a new release shortly.

**Comment by iangrunert** [comment]
Should be fixed in git-ratchet v0.1.4.
